test/api/v3/integration/hall/PUT-hall_heores_heroId.test.js

```diff
@@ -7,6 +7,12 @@ import {
 describe('PUT /heroes/:heroId', () => {
   let user;
 
+  const heroFields = [
+    '_id', 'balance', 'profile', 'purchased',
+    'contributor', 'auth', 'items', 'flags',
+    'secret',
```

Comment:
I like how you've pulled this array out of all the individual tests. Much neater.